.github/workflows/rspec.yml

```
@@ -0,0 +1,14 @@
+name: RSpec
+on: [push, pull_request]
```

nit: I'd like to a line break. Because our OSS is unified with line breaks. Also, please add line breaks before and after to improve readability.

```suggestion

on:
  push:
  pull_request:

```

For your reference, we have included links to our other OSS.

https://github.com/increments/qiita_marker/blob/be76074984c7a30f81856e747ea1f941e5b5def2/.github/workflows/test.yml#L2-L9